public/css/planets.css

```diff
@@ -66,12 +71,154 @@ header {
     height: 35vw;
     border: 1px solid white;
```

Comment:
Instead of the white border there should be a neon pink frame - see the images on our Google drive.